src/components/PageHeader/AudioSetting.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Input, Text, useMatchBreakpoints } from '@pancakeswap-libs/uikit'
+import { useAudioModeManager } from 'state/user/hooks'
+import PancakeToggle from '../PancakeToggle/PancakeToggle'
+
+const StyledAudioSetting = styled.div`
+  margin-bottom: 16px;
+`
+
+const Label = styled.div`
+  align-items: center;
+  display: flex;
+  margin-bottom: 8px;
+`
```

Comment:
Please take a look at https://styled-system.com/

We have the `Box` and `Flex` component from the UI Kit to handle common cases like these.

Reply:
Fixed here and in neighbouring settings modal components 👍